template-processors/base/bin/wrapper.sh

```diff
@@ -17,9 +17,18 @@
 set -euxo pipefail
 
 export HOME=/tmp
-/usr/local/bin/gitClone.sh
-/usr/local/bin/discoverEnvironment.sh
-source $HOME/envs.sh
-/usr/local/bin/processParameters.sh
-/usr/local/bin/processTemplates.sh
-/usr/local/bin/resourceManager.sh
+
+case "$ACTION" in
+  create)
+    /usr/local/bin/gitClone.sh
+    /usr/local/bin/discoverEnvironment.sh
+    source $HOME/envs.sh
+    /usr/local/bin/processParameters.sh
+    /usr/local/bin/processTemplates.sh
+    /usr/local/bin/resourceManager.sh
+    ;;
+  delete)
+    /usr/local/bin/discoverEnvironment.sh
+    /usr/local/bin/resourceManager.sh
+    ;;
```

Comment:
@seanmalloy: ***Important question:*** this PR seems to me to change the behavior for `kubectl delete GitOpsConfig` finalizer when CR has `ResourceHandlingMode: "Patch"`. Question is, was the old behavior in this case even correct? Or, maybe even better question: **What is our intended behavior in this case?** As far as I understand, after this PR, we would effectively do nothing in the finalizer in such case. Previous code seemed to me to try and *delete all the resources listed in the repo*, which is potentially dangerous in my opinion. Personally, I would think doing nothing is the safer choice here, and as such would be OK with this PR's behavior.

Reply:
After some discussion the question is should be deleting resources that eunomia is only patching. :-)

Reply:
We will not delete resources that eunomia has only patched.